lib/test/t_SmolyakExperiment_std.cxx

```diff
@@ -417,27 +417,6 @@ void test_7()
   assert_equal(dimensionBis, 3);
   assert_equal(weightDimensionBis, 83);
   ResourceMap::SetAsBool("SmolyakExperiment-MergeQuadrature", true);
-  // Test 3 : Set tolerances to zero
```

Comment:
I do not understand why this part of the test is removed.
The goal of this part is to show that there are two different ways to prevent the algorithm from merging duplicate nodes:
- either set the "SmolyakExperiment-MergeQuadrature" boolean key to to false (line 404),
- or set the "SmolyakExperiment-MergeRelativeEpsilon" and "SmolyakExperiment-MergeAbsoluteEpsilon" real keys to zero with a "SmolyakExperiment-MergeQuadrature" set to true.

This part of the test shows that the results are the same. Hence, it proves that the management of these ResourceMap keys is correctly performed. If we remove this part of the test, we cannot guarantee that good behaviour anymore. In other words, removing or changing these keys and introducing bugs may go unnoticed in future releases.

Reply:
but this does not work on some architectures because it compares for exactly 0, see #2443

Reply:
Removing the test prevents us from learning something from the fact that the test fails. Let's discuss this further at #2443.